staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -203,6 +207,11 @@ func (s *SecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.PermitPortSharing, "permit-port-sharing", s.PermitPortSharing,
 		"If true, SO_REUSEPORT will be used when binding the port, which allows "+
 			"more than one instance to bind on the same address and port. [default=false]")
+
+	fs.BoolVar(&s.PermitAddressSharing, "permit-address-sharing", s.PermitPortSharing,
```

Comment:
/hold

we also need to fix it to `PermitAddressSharing`

Reply:
the flag would be properly setting `s.PermitAddressSharing` just in the future the default value could be incorrect

Reply:
Good catch!

Reply:
fixed

/hold cancel